protocols/directdefund/directdefund.go

```diff
@@ -210,6 +210,8 @@ func (o *Objective) UpdateWithChainEvent(event chainservice.Event) (protocols.Ob
 				updated.C.OnChainFunding = e.Holdings.Clone()
 			}
 		}
+	case chainservice.ConcludedEvent:
+		break
 	default:
 		return &updated, fmt.Errorf("objective %+v cannot handle event %+v", updated, event)
```

Comment:
Would it be more scalable to have the `default` case simply `break`? 

Here it looks as though we are expecting `ConcludedEvents` to be "routed" to the `directdefund` objective. Is that a sign of a programming error or not? What about some future event type we have yet to cook up and integrate?

Reply:
That's a great question. In this particular case, the `directdefund` protocol might want to observe and record the concluded event. If a client needs to withdraw funds in the unhappy path, it is helpful for the client to know if the channel has been concluded. Though a client can always read the chain state.

In general, if a protocol receives an unexpected event, the protocol should log the event. At the moment, if a protocol receives an unexpected event, this is likely a sign of either a bug or a missing implementation.